### ISSUE TITLE
Android Pie

### ISSUE THREAD

**Issue by dev-abcdevelopers** [beraybentesen/glide-xamarin-android#22]
Image do not load on Android Pie. No error or exception thrown, it just not displaying images at all. I tried it in One Plus 5T.

**Comment by beraybentesen** [comment]
Did you set internet permission?

**Comment by dev-abcdevelopers** [comment]
Yes. I did. It is not loading only in Pie. On other versions it is working quite well.
